src/sentry/api/endpoints/team_projects.py

```diff
@@ -222,4 +222,10 @@ def post(self, request: Request, team: Team) -> Response:
             if project_is_seer_eligible(project):
                 project.update_option("sentry:similarity_backfill_completed", int(time.time()))
 
+            # Add electron symbol server by default to both electron and javascript-electron projects
+            if project.platform and project.platform in ["electron", "javascript-electron"]:
+                project.update_option(
+                    "sentry:builtin_symbol_sources", ["ios", "microsoft", "electron"]
+                )
+
```

Comment:
This does a very similar thing as `set_default_inbound_filters` above. Two suggestions
- Move it up so both of these are placed together. Going forward, this is a good place to add new, similar functionality.
- Create a helper function that you can also call from the `DatabaseBackedProjectService`.

Reply:
Yeah I was pondering that, but then we set the seer project option `sentry:similarity_backfill_completed` below, so I followed that lead. But good points, rearranged and pulled it into a helper function.